packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+import 'dart:ui' as ui;
+
+import 'package:flutter/services.dart';
+import 'package:mojo/bindings.dart' as mojo_bindings;
+import 'package:mojo/core.dart' as mojo_core;
+import 'package:sky_services/pointer/pointer.mojom.dart';
+
+import 'arena.dart';
+import 'converter.dart';
+import 'events.dart';
+import 'hit_test.dart';
+import 'pointer_router.dart';
+
+abstract class Pointerer extends BindingBase implements HitTestTarget, HitTestable {
```

Comment:
Yeah, Pointerer was my placeholder name. I was thinking maybe Gesturer?

I'm trying to keep a consistent naming scheme between the various binding mixins. There's one in the scheduler library, one in the gestures library, one in the rendering library, and maybe one in the widgets library, though at the moment I'm not using a mixin for the last one, that's a concrete class. (There's also a concrete class at the rendering level though that one just brings in the mixins without additional logic involved.)